### DISCUSSION TITLE
[Feature?]: A different naming system for "route group"

### DISCUSSION THREAD

**Discussion by xarthurx** [solidjs/solid-start#1150] [Ideas]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

In https://start.solidjs.com/core-concepts/routing#route-groups, it explains the naming of a route group is to use `()` in the folder name.

This can create some issues for text editors on different platform (eg https://github.com/neovim/neovim/issues/24542) due to the "escape" effect.

Would it be considerable if using something like `__foldername__` or `--foldername--` rather than the `()`?

**Comment by JAD3N** [comment]
It does also seem a bit strange that it matches the syntax for [renaming index](https://start.solidjs.com/core-concepts/routing#renaming-index).
